web/src/components/ServiceList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Card, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        maxWidth: 400,
+        marginLeft: 10,
+        marginTop: 10,
+    },
+    title: {
+        fontSize: 14,
+    },
+    details: {
+        margin: '0 2px',
+    },
+    cardView: {
+        padding: theme.spacing(2),
+    },
+}));
+
+const ServiceList = () => {
+    const [services, setServices] = useState(null);
+    const classes = useStyles();
+    const fetchData = async () => {
```

Comment:
this function could be named better, maybe like getServices or fetchServices?

Reply:
ill change it to GetServices but notice the capital on Get. React components must start with a capital.